### ISSUE TITLE
member part contradiction

### ISSUE THREAD

**Issue by fcbr** [ontologyportal/sumo#8]
Very similar to #7:

```
(subrelation member part)
(instance member IntransitiveRelation)
```

and:

```
(instance part PartialOrderingRelation)
(instance part SpatialRelation)
```

and:

```
(subclass SpatialRelation InheritableRelation)
(subclass PartialOrderingRelation TransitiveRelation)
```

so we deduce:

```
(instance member TransitiveRelation)
```

but:

```
(disjoint TransitiveRelation IntransitiveRelation)
```

So the contradiction is similar to #7, where `member` is both `IntransitiveRelation` and `TransitiveRelation`.

**Comment by apease** [comment]
subrelation statement was removed in a previous version.  Closing issue

**Comment by arademaker** [comment]
Solved in 89d8923.
